docs/guides/parallel-scraping/parallel-scraping.mdx

```diff
@@ -0,0 +1,209 @@
+---
+id: parallel-scraping-guide
+title: Parallel Scraping Guide
+sidebar_label: Parallel Scraping
+description: Parallelizing your scrapers with Crawlee
+---
+
+import ApiLink from '@site/src/components/ApiLink';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import CodeBlock from '@theme/CodeBlock';
+
+import SharedSource from '!!raw-loader!./shared.mjs';
+import AdaptedRoutesSource from '!!raw-loader!./adapted-routes.mjs';
+import ParallelScraperSource from '!!raw-loader!./parallel-scraper.mjs';
+import ModifiedDetailRouteSource from '!!raw-loader!./modified-detail-route.mjs';
+
+:::warning Experimental features ahead
+
+At the time of writing this guide (November 2023), request locking is still an experimental feature. You can read more about the experiment by visiting the [request locking experiment](../experiments/experiments-request-locking) page.
+
+:::
+
+In this guide, we will walk through how you can turn your single scraper into a scraper that can be parallelized and run in multiple instances. This guide
+assumes you've read and walked through our [introduction guide](../introduction/setting-up) (or have a fully-fledged scraper already built), but if you haven't done so yet,
+take a break, go read through all that, and come back. We'll be waiting...
+
+*Oh, you're back already! Let's proceed in making that scraper parallel!*
+
+## Things to consider before parallelizing
+
+Before you rush ahead and change your scraper to support parallelization, take a minute to consider the following factors:
+
+- do you plan on scraping so many pages that you need to parallelize your scraper?
```

Comment:
Is there any preference for use of - instead of * for unordered lists? I haven't seen it documented before so I added in style guide to use * and have been following this rule with all docs created since, but that can be easily changed and fixed

Reply:
I think we use `-` usually, at least that's what I do as it's just one key away, unlike `*` which requires `shift` too :] So if we are picking one, I'd prefer `-` over `*` personally.

Let's not care about this now, we should add some code formatting that will fix this automatically. Let's first do it in the apify docs, then we can port it here.

Reply:
I'll remove then the mention for this from style guide for now